pages/core-api/safe-installation-guide.mdx

```diff
@@ -0,0 +1,250 @@
+import { Callout, Steps, Tabs } from 'nextra/components'
+
+# Safe stack installation guide
+
+## Safe\{Core\} contracts deployment
+
+In the first step, you will deploy the Safe\{Core\} contracts on your chain.
+
+All Safe contract deployments on any network follow the same procedure to ensure a deterministic address for all singleton contracts (proxy-factory, mastercopy, etc.) and verify the deployment.
+
+### Prerequisites
+
+Open a [pull request](https://github.com/ethereum-lists/chains) to add your chain to [chainlist.org](https://chainlist.org/).
+
+### Steps
+
+<Steps>
+
+### Singleton factory contract deployment
+
+<Callout type="info" emoji="ℹ️">
+ You do not need to perform these tasks if your network is based on a rollup framework with Safe contracts already deployed (for example, OP Stack).
+</Callout>
+
+1. Create a new issue in the [safe-singleton-factory](https://github.com/safe-global/safe-singleton-factory/issues/new?assignees=&labels=&projects=&template=new_chain.yml&title=%5BNew+chain%5D%3A+) repository.
+2. A bot will reply to the issue with the deployer address (`0x914...3d7`) and the amount of native token you need to send to this address.
+3. Once funded, mark the checkbox on the GitHub issue.
+4. The review of the issues happens every two weeks. Our team will perform the deterministic deployment of the `safe-singleton-factory` contract and publish a new npm release of [@safe-global/safe-singleton-factory](https://www.npmjs.com/package/@safe-global/safe-singleton-factory).
+
+### Singleton contracts deployment 
+
+<Callout type="info" emoji="ℹ️">
+ You do not need to perform these tasks if your network is based on a rollup framework with Safe contracts already deployed (for example, OP Stack).
+</Callout>
+
+1. Clone the [safe-smart-account](https://github.com/safe-global/safe-smart-account) repository by running the following command:
+    ```
+    git clone --branch v1.3.0-libs.0 https://github.com/safe-global/safe-smart-account.git
+    ```
+    
+2. Get the latest version of [@safe-global/safe-singleton-factory](https://www.npmjs.com/package/@safe-global/safe-singleton-factory), by running the following command:
+
+    ```bash
+    npm i --save-dev @safe-global/safe-singleton-factory
+    ```
+
+    Ensure the latest version includes your `safe-singleton-factory` deployment from the earlier section.
```

Comment:
Let's add a (relative) link to this section: #singleton-factory-contract-deployment